requirements.txt

```diff
@@ -1,13 +1,16 @@
 Django>=4
 djangorestframework==3.13.1
+django-filter>=2.4
 psycopg2-binary==2.9.3
 django-bootstrap4==22.1
+fastavro>=1.7
 whitenoise==6.0.0
 lcogt-logging==0.3.2
 hop-client~=0.7
+healpy>=1.16
+tom_alertstreams==0.5.2
```

Comment:
I only see `tom_alertstreams==0.5.1` on pypi. 
Has 0.5.2 been deployed yet?

Reply:
hmm your right... I thought I had deployed it but looks like the action didn't run on tag. I have another PR for it now so I'll just try again with version 0.5.3